Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentController.h

```diff
@@ -95,5 +96,6 @@ namespace MiniAudio
 
         AZStd::unique_ptr<ma_sound> m_sound;
         AZStd::string m_soundName;
+        AZ::u64* m_cursor;
```

Comment:
Where is this member variable used?

Reply:
Oops, it's not, I was testing something else and this accidentally carried over. Just took it out.